packages/open-payments/src/client/ilp-stream-connection.test.ts

```diff
@@ -15,14 +28,34 @@ describe('ilp-stream-connection', (): void => {
   const axiosInstance = defaultAxiosInstance
   const logger = silentLogger
 
-  describe('createILPStreamConnectionRoutes', (): void => {
-    test('calls createResponseValidator properly', async (): Promise<void> => {
-      jest.spyOn(openApi, 'createResponseValidator')
+  describe('routes', (): void => {
+    const ilpStreamConnection = mockILPStreamConnection()
+
+    describe('get', (): void => {
+      test('calls get method with correct validator', async (): Promise<void> => {
+        const mockResponseValidator = ({ path, method }) =>
+          path === '/connections/{id}' && method === HttpMethod.GET
+
+        jest
+          .spyOn(openApi, 'createResponseValidator')
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
+          .mockImplementation(mockResponseValidator as any)
```

Comment:
Could we check that the spy is called?

Reply:
It's already testing this (albeit indirectly): if the `createResponseValidator` doesn't get the proper `path` and `method` when, the test will fail based on the expectation of `true` on L57

Reply:
Ah, the `true`! :+1:

Reply:
It's not the easiest to see when first reading it, but that was the best solution I could come up with to test the correct validator was passed in